mods/creative/mod.conf

```diff
@@ -1,3 +1,4 @@
 name = creative
 description = Minetest Game mod: creative
 depends = default, sfinv
+optional_depends = beds
```

Comment:
why is this needed?

Reply:
I mentioned in the original post. I hate having it there but I don't know why it doesn't load otherwise.

Reply:
As in, it doesn't show up in the creative inventory if I don't put it as an optional dependency.
The only reason I can think of is, beds is loaded first as the mods are loaded alphabetically.

Reply:
maybe you should read the documentation? 
https://github.com/minetest/minetest/blob/6d8e2d2483715f05216be86a32f664bcbaac3efe/doc/lua_api.txt#L4217

Reply:
That works; thanks :smiley: